graspy/inference/latent_distribution_test.py

```diff
@@ -137,24 +112,27 @@ def fit(self, A1, A2):
         p_ : float
             The p value corresponding to the specified hypothesis test
         """
-        A1 = import_graph(A1)
-        A2 = import_graph(A2)
-        if not is_symmetric(A1) or not is_symmetric(A2):
-            raise NotImplementedError()  # TODO asymmetric case
-        if A1.shape != A2.shape:
-            raise ValueError("Input graphs do not have matching dimensions")
-        if self.n_components is None:
-            # get the last elbow from ZG for each and take the maximum
-            num_dims1 = select_dimension(A1)[0][-1]
-            num_dims2 = select_dimension(A2)[0][-1]
-            self.n_components = max(num_dims1, num_dims2)
-
-        X1_hat, X2_hat = self._embed(A1, A2)
-        X1_hat, X2_hat = self._median_heuristic(X1_hat, X2_hat)
-        U = self._statistic(X1_hat, X2_hat)
-        null_distribution = self._bootstrap(X1_hat, X2_hat, self.n_bootstraps)
-        self.null_distribution_ = null_distribution
-        self.sample_T_statistic_ = U
-        p_value = (len(null_distribution[null_distribution >= U])) / self.n_bootstraps
-        self.p_ = p_value
+        if self.graph:
+            A1 = import_graph(A1)
+            A2 = import_graph(A2)
+            if not is_symmetric(A1) or not is_symmetric(A2):
+                raise NotImplementedError()  # TODO asymmetric case
+            if self.n_components is None:
+                num_dims1 = select_dimension(A1.astype(float))[0][-1]
+                num_dims2 = select_dimension(A2.astype(float))[0][-1]
+                self.n_components = max(num_dims1, num_dims2)
+            X1_hat, X2_hat = self._embed(A1, A2)
+            X1_hat, X2_hat = self._k_sample_transform(X1_hat, X2_hat)
+        else: #you already gave me stuff
+            X1_hat = A1
```

Comment:
if you want to allow the passing of embeddings please explain this and the `graph` kwarg in the docs